### PR TITLE
button menu's internal state cannot be overriden. work with a copy.

### PR DESCRIPTION
**Pull request by nullhook**

### CODE REVIEW

File: src/components/buttonMenu/buttonMenu.svelte [hunk 1]
```diff
@@ -40,7 +41,7 @@
     <slot name="anchor-content">Click</slot>
   </div>
   <Menu
-    bind:isOpen
+    bind:isOpenInternal
```
Comment (fallaciousreasoning) [inline src/components/buttonMenu/buttonMenu.svelte:44]: I don't think you can bind `isOpenInternal` like this - what's it binding to?

Comment (fallaciousreasoning) [reply src/components/buttonMenu/buttonMenu.svelte:44]: Additionally, maybe we shouldn't use bind here at all? What if the API on Menu changes and it doesn't emit an onClose - `isOpenInternal` would change but our `isOpen` state wouldn't get updated (and we might not emit our close event).

Comment (nullhook) [reply src/components/buttonMenu/buttonMenu.svelte]: fixed

**Review by fallaciousreasoning — APPROVED** [review]
Looks good @nullhook (assuming you have it working in brave-core) - mind squashing when you commit and adding a `[ButtonMenu]: ` prefix, so we know what component this affected in the changelog.

**Pull request merged**